### DISCUSSION TITLE
Elli Charger Pro

### DISCUSSION THREAD

**Discussion by Exo84** [camueller/SmartApplianceEnabler#564] [Allgemein]
Hallo,

Weiß jemand ob ich del SAE mit dem Elli Charger Pro verbinden könnte?

Gruß

**Comment by camueller** [comment]
Die Wallbox ist nur per EEBus und OCPP steuerbar. Beides unterstützt der SAE nicht.
Allerdings kann der Sunny Home Manager EEBus, d.h. möglicherweise kannst Du die Wallbox direkt mit dem Sunny Home Manager verbinden.

**Reply by Exo84** [reply]
Danke
